src/reducers/index.ts

```diff
@@ -18,9 +42,64 @@ function reducer(state: Readonly<StateBase>, action: Action): StateBase {
   };
 }
 
+/**
+ * Exclude these actions from the history completely.
+ */
+const excludeActionsInHistory = [DATASET_INLINE_RECEIVE,
```

Comment:
2,3: For _DATASET_INLINE_RECEIVE_. At the moment that is not something a user action can trigger, so I don't think it should be in the history at all (Though co-incidentally  it will be wiped from the history once the action is complete anyway). Same reason for SET_CONFIG (minus the auto clearing of history). I can add a comment to this effect. However if you can think of reasons why they should be in the history, let me know.

4. +1 on line break